httsleep/main.py

```diff
@@ -84,6 +87,9 @@ def __init__(self, url_or_request, until=None, alarms=None,
                          'jsonpath': jsonpath, 'text': text, 'callback': callback}
             until.append({k: v for k, v in condition.items() if v})
 
+        self.kwargs = {}
+        if verify is not None:
```

Comment:
This will now never happen.. I'd be fine with simply always passing `verify` to requests as a kwarg on L154 (no need for `self.kwargs`, which is a little ambiguous)